modules/databases/postgresql_flexible_server/server.tf

```diff
@@ -18,7 +18,8 @@ resource "azurerm_postgresql_flexible_server" "postgresql" {
   storage_mb          = try(var.settings.storage_mb, null)
 
   delegated_subnet_id = var.remote_objects.subnet_id
-  private_dns_zone_id = var.remote_objects.private_dns_zone_id
+  # private_dns_zone_id = var.remote_objects.private_dns_zone_id
```

Comment:
If this is the final working version we should remove old code, not leave it commented, or things will get very messy